### DISCUSSION TITLE
How to push a button before values are displayed?

### DISCUSSION THREAD

**Discussion by kamicater** [jomjol/AI-on-the-edge-device#1341] [General]
Modern water meters have an LC-Display showing nothing at all. After pushing a button (usually round and black) the numbers appear. How could one solve this challenge?

![grafik](https://user-images.githubusercontent.com/9930740/201524727-a17d163f-fd92-40ab-8f32-3ab4b379a380.png)

**Comment by caco3** [comment]
The ESP32-CAM has some spare GPIOs which could be used for this 8like for controlling an external LED controller.

How ever, the real challenge for you is how to push the button. This either needs some mechanical pusher or you rip it apart and short circuit the button on request.

**Reply by kamicater** [reply]
The smart home community is so huge and enthusiastic. I'm pretty sure somewhere someone will come up with a great idea. That someone is definetly not me 😁
